lib/account.php

```diff
@@ -92,6 +93,16 @@ public function getImapConnection() {
 			if ($this->config->getSystemValue('app.mail.imaplog.enabled', false)) {
 				$params['debug'] = $this->config->getSystemValue('datadirectory') . '/horde.log';
 			}
+			if ($this->config->getSystemValue('app.mail.server-side-cache.enabled', false)) {
+				$cacheFac = \OC::$server->getMemCacheFactory();
```

Comment:
Eww... can we not use dependency injection for this? Failing that, move this line to the constructor, where `$this->config` et al are set?